insights/parsers/ps.py

```diff
@@ -104,22 +105,76 @@ def users(self, proc):
         for row in self.data:
             if proc == row["COMMAND"]:
                 if row["USER"] not in ret:
-                    ret.update({row["USER"]: []})
+                    ret[row["USER"]] = []
                 ret[row["USER"]].append(row["PID"])
         return ret
 
     def fuzzy_match(self, proc):
-        """boolean: Returns ``True`` if ``proc`` is in the COMMAND column."""
-        return proc in "".join(self.running)
+        """boolean: Returns ``True`` if the word ``proc`` appears in the COMMAND column."""
+        return any(proc in row['COMMAND'] for row in self.data)
 
     def __contains__(self, proc):
-        return proc in self.running
+        # We should be able to take this if out when we only have the PsAuxww
+        # parser:
+        return any(proc == row['COMMAND'] for row in self.data if 'COMMAND' in row)
 
     def __iter__(self):
         for row in self.data:
             yield row
 
 
+@parser('ps_auxww')
+class PsAuxww(ProcessList):
+    """
+    Class to parse ``ps auxww`` command output.
+
+    Examples:
```

Comment:
Do you need to add the examples?

Reply:
The examples are in the headings, so I've removed this.